webapp/components/Upload/index.vue

```diff
@@ -44,6 +45,14 @@ export default {
       }
     },
   },
+  watch: {
+    error() {
+      let that = this
```

Comment:
Was this not solved with EcmaScript? :thinking: could you try out with `error: () => {` arrow functions?

Reply:
Or, what is `this ` actually here? Could you explain? :thinking:

Reply:
I tried it out and it seems to work, but it breaks the tests, so maybe it doesn't or maybe the tests need to be modified somehow... to be honest, we copied and pasted this from the `vue2-dropzone` demo...